src/components/tutorial-collection-cards/components/card-heading/card-heading.module.css

```diff
@@ -0,0 +1,28 @@
+.root {
+  composes: hds-typography-body-200 from global;
+  color: var(--token-color-foreground-strong);
+  font-weight: var(--token-typography-font-weight-semibold);
+
+  /* Spacing for heading text */
+  margin: 0 0 4px 0;
+
+  &:not(:first-child) {
+    margin-top: 8px;
+  }
+
+  /* Spacing for heading logo */
+  &.hasLogo {
+    margin-bottom: 8px;
+
+    &:not(:first-child) {
+      margin-top: 16px;
+    }
+  }
+}
+
+.logo {
+  display: block;
+  height: 20px;
```

Comment:
Some of the logos look a little off, small. But we probably just need to adjust the whitespace maybe?

Reply:
Great call - just flagged this with Brayden. I think since our `Terraform Cloud` logo is coming from a blessed repo, while all the others are local `.png`s, the easiest path forward is to make em all look the same size with some top & bottom whitespace built into said PNGs.

I'll plan to patch this up in a follow-up PR, after the collection views are done... or during VQA, I would be surprised if it doesn't come up again at that point if we do forget to do it (feels like it's lower priority, but lemme know if it feels more critical to address 👍).

Reply:
Perfect, definitely lower priority for sure.